README.md

```diff
@@ -738,20 +745,27 @@ class AsyncDisposableStack {
 
   /**
    * Adds a resource to the top of the stack.
```

Comment:
"Adds a disposable resource to the top of the stack. Has no effect if given null or undefined."?